computations.py

```diff
@@ -0,0 +1,140 @@
+import calendar
+import sys
+
+
+class Computations:
+
+    @staticmethod
+    def get_weather_attribute_index(weather_attributes_list, attribute):
+        return weather_attributes_list[attribute]
+
+    @staticmethod
+    def get_month_and_day_from_date(date):
+        year, month, day = date.split("-")
+        return int(month), int(day)
+
+    @staticmethod
+    def find_maximum_temperature_in_year(year, all_weather_records):
+        weather_attributes, weather_records = all_weather_records
+        maximum_temperature = Computations.get_weather_attribute_index(weather_attributes, "Max TemperatureC")
+        date = Computations.get_weather_attribute_index(weather_attributes, "PKT")
+        maximum_temperature_in_year = float("-inf")
+        maximum_temperature_record_in_year = []
+        try:
+            for weather_monthly_records in weather_records[year].keys():
+                weather_daily_records = weather_records[year][weather_monthly_records]
+                maximum_temperature_record_in_month = max(
+                    weather_daily_records, key=lambda x: x[maximum_temperature] if x[maximum_temperature] is not None
+                    else float('-inf'))
```

Comment:
Too much going on in one-liner, this compromises readability too much. Lets remove conditional part from labda.